src/components/content/conditions/patient-conditions.tsx

```diff
@@ -1,130 +1,68 @@
-    className,
-    readOnly = false,
-    hideBuilderOwnedRecords = false,
-    hideOutsideOwnedRecords = false,
-    hideRequestRecords = false,
-  }: PatientConditionsProps) => {
-    // State.
-    const [collection, setCollection] = useState<FilterCollection>(
-      hideBuilderOwnedRecords ? "other" : "patient"
-    );
-    const { filters, updateFilters, applyFilters } =
-      useConditionFilters(collection);
-    const { applySorts, sortOptions, updateSorts, currentSorts } =
-      useConditionSorts(collection);
-    const containerRef = useRef<HTMLDivElement>(null);
-    const breakpoints = useBreakpoints(containerRef);
-
-    // Drawer helpers.
-    const showConditionHistory = useConditionHistory();
-
-    // Data fetching.
-    const patientConditionsQuery = usePatientConditions();
-    const otherConditionsQuery = useOtherProviderConditionsDeduped();
+export const PatientConditions = ({ className }: PatientConditionsProps) => {
+  const query = usePatientConditions();
+  const showAddConditionForm = useAddConditionForm();
 
-    function isLoading() {
-      const isLoadingPatient = patientConditionsQuery.isLoading;
-      const isLoadingOther = isLoadingPatient || otherConditionsQuery.isLoading;
-      return collection === "patient" ? isLoadingPatient : isLoadingOther;
-    }
+  return (
+    <PatientConditionsBase
+      query={query}
+      className={className}
+      action={
+        <button
+          type="button"
+          className="ctw-btn-primary ctw-p-0"
+          onClick={() => showAddConditionForm()}
+        >
+          Add Condition
+        </button>
+      }
+      rowActions={RowActions}
+    />
+  );
+};
 
-    // Get our conditions.
-    const patientConditions = patientConditionsQuery.data ?? [];
-    const otherConditions = otherConditionsQuery.data;
+const RowActions = ({ record }: RowActionsProps<ConditionModel>) => {
+  const showEditConditionForm = useEditConditionForm();
+  const confirmDelete = useConfirmDeleteCondition();
 
-    const conditions = applySorts(
-      applyFilters(patientConditions, otherConditions)
-    );
-    const RowActions =
-      collection === "patient"
-        ? PatientConditionHoverActions
-        : OtherProviderConditionHoverActions;
+  return (
+    <div className="ctw-flex ctw-space-x-2">
+      {!record.isDeleted && (
+        <button
+          type="button"
+          className="ctw-btn-default"
+          onClick={(event) => {
+            event.stopPropagation();
+            confirmDelete(record);
+          }}
+        >
+          Remove
+        </button>
+      )}
 
-    return (
-      <div
-        ref={containerRef}
-        className={cx("ctw-patient-conditions ctw-bg-white", className, {
-          "ctw-patient-conditions-stacked": breakpoints.sm,
-        })}
+      <button
+        type="button"
+        className="ctw-btn-primary"
+        onClick={(event) => {
+          event.stopPropagation();
+          showEditConditionForm(record);
+        }}
```

Comment:
If my last comment about the action buttons event propagation is put into code then this also would become
```suggestion
        onClick={() => showEditConditionForm(record)}
```